docs/site/Project-layout-reference.md

```diff
@@ -0,0 +1,123 @@
+---
+lang: en
+title: 'Project Layout Reference'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Project-layout-reference.html
+---
+
+LoopBack 4 project files and directories are in the _application root
+directory_. Within this directory the standard LoopBack 4 project structure has
+these sub-directories:
```

Comment:
We should mention the project layout was created by `lb4 application` command and new artifacts can be added by other `lb4` commands or manually by application developers.

Reply:
👍 I added CLI links to each artifact.